src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenADependencyContextBuilder.cs

```diff
@@ -28,7 +28,9 @@ public class GivenADependencyContextBuilder
             CompilationOptions compilationOptions,
             string baselineFileName,
             string runtime,
-            ITaskItem[] satelliteAssemblies)
+            ITaskItem[] assemblySatelliteAssemblies,
```

Comment:
Agreed, it is a poor name.  But it matches the name in the product (GenerateDepsFile class).  And the reason it is named that in the product is because it matches all the other properties:

- AssemblyName
- AssemblyExtension
- AssemblyVersion
- AssemblySatelliteAssemblies